### DISCUSSION TITLE
Kriegsspiel game mode

### DISCUSSION THREAD

**Discussion by ddugovic** [woogles-io/liwords#853] [Ideas]
In Kriegsspiel (chess) it is possible to see your pieces, but not your opponent's pieces.

For OMGWords I can imagine a couple game modes (VOID challenge):
* Opponent's tiles are invisible
* Opponent's tiles are visible but the letters are invisible

Either way, the score might provide some clue as to what they played.

**Comment by erwingame** [comment]
I also want to know.
